services/whos_on_duty.rb

```diff
@@ -13,7 +13,11 @@ def list
     devs          = Builder::Dev.hash(source.devs(:current), source.devs(:next))
     duty_managers = Builder::Manager.hash(fetch_managers)
 
-    webops + devs + duty_managers
+    unless webops.any? && devs.any? && fetch_managers == pagerduty_managers
+      []
```

Comment:
It's not clear what this is doing.  At a guess, I would say it's returning an empty array if the webops array is empty and the devs array is empty and fetch managers  is not equal to pagerduty managers, is that correct?

if so, might be more readable as:

if webops.empty? && devs.empty? && fetch_managers != pagerduty_managers